guides/routing/htmlforms.html

```diff
@@ -0,0 +1,246 @@
+    <p class="sentence">2. Open Sources > Application > Application.swift</p>
+    <p class="sentence">3. Inside the postInit function, add the static file server middleware: </p>
+    <pre><code class="language-swift">router.get("/", middleware: StaticFileServer())</code></pre>
+    <p class="sentence">4. In the top left of Xcode is a toolbox icon. Click this and change the scheme from "HTMLForms-Package" to "HTMLForms".</p>
+    <p class="sentence">5. Run the server and go to: <a href="http://localhost:8080/formwebpage.html" target="_blank">localhost:8080/formwebpage.html</a></p>
+    <p class="sentence">You should see a simple HTML page with four HTML forms. These will send the input data to our server. Now we need to create the server routes that receive this data.</p>
+
+    <h2><a name="form-model" style="padding-top: 100px;"></a>Create the Form Model</h2>
+
+    <p class="sentence">The Kitura server works directly with Swift value types, therefore we need to contruct a value type to use.
+    <p class="sentence">It's good practice to keep value types (in this case models) and logic separated. Create a new file, to contain the models, called Models.swift as follows:</p>
+    <ul>
+      <li>1. In Xcode right click on the Application folder and select New File.</li>
+      <li>2. Select Swift File in the UI pop up and click next.</li>
+      <li>3. In the Save As field, name the file Models.swift.</li>
+      <li>4. Ensure Group is Application and target is Application.</li>
+      <li>5. Click Create to create the file.</li>
+    </ul>
+    <p class="sentence">For now, let's create a very simple model:</p>
+    <p class="sentence">1. Open Models.swift</p>
+    <p class="sentence">2. Copy in the following code:</p>
+    <pre><code class="language-swift">import KituraContracts
+
+struct Form: Codable, QueryParams {
+    let name: String
+    let age: Int?
+    let isDeveloper: Bool
+}</code></pre>
+    <p class="sentence">This provides a simple representation of a data we will recieve from our form. We marked it as Codable so it can be used in Codable routes and as QueryParams so it can be decoded from query parameters on a GET request.</p>
+    <p class="sentence">If the forms "age" field is empty, it will send the empty string in the query and Kitura will decode the value to nil. If the forms "isDeveloper" field is empty, it will send the empty string in the query and Kitura will decode the value to false.
+    <p class="sentence">Now we have a model we can use, we can create our routes!</p>
+
+
+    <h2><a name="codable-routes" style="padding-top: 100px;"></a>Setup the Codable Routes</h2>
+    <p class="sentence">We will now create a <a href="https://www.kitura.io/guides/routing/codablerouting.html target="_blank"">Codable route</a> to recieve the POST request from an HTML form.</p>
+    <p class="sentence">1. Open Application.swift</p>
+    <p class="sentence">2. Inside the "postInit" function, Register the route handler:</p>
```

Comment:
You  need to search the document for postInit references - some are currently "postInit" and some are postInit - we should be consistent.  Ideally we would have a style we could apply to function names.

We also have ", Register" throughout the document which needs changing to ", register"

Reply:
The first part of this is fixed - however there are still references to ", Register" in the document which need fixing.